auto_round/script/llm.py

```diff
@@ -275,7 +275,9 @@ def tune(args):
         if format not in supported_formats:
             raise ValueError(f"{format} is not supported, we only support {supported_formats}")
         if format in ["gguf:q4_0", "gguf:q4_1"]:
-            args.bits = 4
+            if args.bits != 4:
+                args.bits = 4
```

Comment:
1 merge all the related warning into one. 
2 If only gguf is exported, log warning and force change, if there are multiple formats to export, log error and exit